travis/travis_install_nightly

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-pip install -q QUnitSuite flake8 coveralls pylint Click > /dev/null 2>&1
+pip install -q QUnitSuite flake8 coveralls Click oca-pylint-plugin > /dev/null 2>&1
```

Comment:
IMHO if our wait to create a new project, review of big code, refactories, new discuss... and later merge this one the process will be many many slow.
We are using other libraries as flake8, coveralls, pylint now then use oca-pylint-plugin is similar to use other more.

Then If our merge this one the creation of a new project to oca-pylint-pluing will make more sense that before merge.